### DISCUSSION TITLE
workflow: Notify users based on actions from others users

### DISCUSSION THREAD

**Discussion by AmadeuAntunes** [OrchardCMS/OrchardCore#8291] [Q&A]
Hello,
I'm trying to use workflows to create a way to set a approval workflow.
Is there any way to notify administrators that some user edited a content without send emails ?

**Comment by jsalgado10** [comment]
What type of notifications are looking for?

I am working on a notification module that will email users on new content post or updates. I have also added a custom controller to allow users to submit content and save it as a draft. The notification module will email administrators so they can review and publish the content.

**Reply by Skrypt** [reply]
Use a Workflow for that.

**Reply by jsalgado10** [reply]
Besides sending notifications to admin, I would like to send notifications on new Published content to other users.
I added some custom settings to the user profile to turn on/off notifications. Is there a way on Workflow to check for those values and send the email to only the user that have opt-in?

**Reply by Skrypt** [reply]
Sure, you can do DB Queries within a Workflow execution. Here you could easily set up a SQL Query that does a SELECT on your custom user index.
